src/unix/mod.rs

```diff
@@ -228,6 +228,11 @@ cfg_if! {
         #[link(name = "root")]
         #[link(name = "network")]
         extern {}
+    } else if #[cfg(target_os = "fuchsia")] {
+        #[link(name = "c")]
+        #[link(name = "mxio")]
+        #[link(name = "unwind")]
```

Comment:
Ah yeah note that if it's removed here it'll need to be added to `src/unwind/build.rs` most likely (as that's where the unwinder symbols are defined